deploy/helm/kubecf/assets/operations/instance_groups/app-autoscaler.yaml

```diff
@@ -0,0 +1,1049 @@
+      domain: bosh
+      instance_group: asnozzle
+      network: default
+      query: '*'
+- type: replace
+  path: /addons/name=bosh-dns-aliases/jobs/name=bosh-dns-aliases/properties/aliases/-
+  value:
+    domain: metricsserver.service.cf.internal
+    targets:
+    - deployment: {{ .Release.Name | quote }}
+      domain: bosh
+      instance_group: asmetrics
+      network: default
+      query: '*'
+
+# Add autoscaler related releases.
+- type: replace
+  path: /releases/-
+  value:
+    name: app-autoscaler
+    version: {{ (index .Values.releases "app-autoscaler").version | quote }}
+    url: {{ include "kubecf.releaseURLLookup" (list .Values.releases "app-autoscaler") }}
+    stemcell: {{ include "kubecf.stemcellLookup" (list .Values.releases "app-autoscaler") }}
+- type: replace
+  path: /releases/-
+  value:
+    name: postgres
+    version: {{ .Values.releases.postgres.version | quote }}
+    url: {{ include "kubecf.releaseURLLookup" (list .Values.releases "postgres") }}
+    stemcell: {{ include "kubecf.stemcellLookup" (list .Values.releases "postgres") }}
+
+# Add autoscaler instance groups.
+- type: replace
+  path: /instance_groups/0:before
+  value:
+    name: postgres_autoscaler
+    persistent_disk: 20480 # 20GB
+    {{- if .Values.kube.storage_class }}
+    persistent_disk_type: {{ .Values.kube.storage_class }}
+    {{- end }}
+    azs:
+    - z1
+    instances: 1
+    stemcell: default
+    networks:
+    - name: default
+    jobs:
+    - name: postgres
+      release: postgres
+      properties:
+        databases:
+          tls:
+            ca: ((postgres_ca.certificate))
+            certificate: ((postgres_server.certificate))
+            private_key: ((postgres_server.private_key))
+          databases:
+          - name: autoscaler
+            tag: default
+          port: &postgresPort 5432
+          roles:
+          - name: postgres
+            password: "((autoscaler_database_password))"
+            tag: default
+- type: replace
+  path: /instance_groups/-
+  value:
+    name: asactors
+    azs:
+    - z1
+    instances: 1
+    networks:
+    - name: default
+    stemcell: default
+    update:
+      max_in_flight: 1
+      serial: true
+    jobs:
+    - name: scalingengine
+      release: app-autoscaler
+      properties:
+        autoscaler:
+          scalingengine_db: &database
+            databases:
+            - name: autoscaler
+              tag: default
+            db_scheme: postgres
+            port: 5432
+            roles:
+            - name: postgres
+              password: ((autoscaler_database_password))
+              tag: default
+            sslmode: verify-full
+            tls:
+              ca: ((postgres_ca.certificate))
+          scheduler_db: *database
+          policy_db: *database
+          scalingengine_db_connection_config: &databaseConnectionConfig
+            max_open_connections: 100
+            max_idle_connections: 10
+            connection_max_lifetime: 60s
```

Comment:
Still find it odd that the `app-autoscaler-release` doesn't have reasonable default values for these, but I'm okay with leaving them here.